### ISSUE TITLE
Fix incorrect role for header

### ISSUE THREAD

**Issue by neeleshb** [microsoft/vscode#190460]
Type: <b>Bug</b>

The header row containing the menubar and banner has the role "contentinfo" which is incorrect because the "contentinfo" role is reserved only for the footer.  The role should be removed (changed to none / presentation). 

VS Code version: Code 1.78.0 (252e5463d60e63238250799aef7375787f68b4ee, 2023-05-03T20:09:46.685Z)
OS version: Linux x64 6.3.11-1rodete1-amd64
Modes:
Sandboxed: No

**Comment by VSCodeTriageBot** [comment]
Thanks for creating this issue! It looks like you may be using an old version of VS Code, the latest stable release is 1.81.1. Please try upgrading to the latest version and checking whether this issue remains.

Happy Coding!
